src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -22,7 +22,7 @@
     <script asp-name="font-awesome-v4-shims" at="Head" version="6"></script>
     <script asp-name="js-cookie" at="Foot"></script>
     <script asp-name="admin" version="1" at="Foot"></script>
-    <resources type="Header" />
+    <resources type="Header"></resources>
```

Comment:
Why is this change necessary? I hoe `<resources>` doesn't end up in the HTML output though? Same for the Footer.